packages/ra-ui-materialui/src/input/ArrayInput/ArrayInput.tsx

```diff
@@ -156,36 +199,32 @@ export const ArrayInput = (props: ArrayInputProps) => {
             margin={margin}
             className={clsx(
                 'ra-input',
-                `ra-input-${source}`,
+                `ra-input-${finalSource}`,
                 ArrayInputClasses.root,
                 className
             )}
             error={!!error}
             {...sanitizeInputRestProps(rest)}
         >
             <InputLabel
-                htmlFor={finalSource}
```

Comment:
why did you remove that?

Reply:
Because you can't label a non focusable element